tools/metricsdocs/metricsdocs.go

```diff
@@ -49,33 +49,61 @@ func metricsOptsToMetricList(metrics map[monitoring.MetricsKey]monitoring.Metric
 	return result
 }
 
-func readFromPrometheusCR() metricList {
-	data := make(map[interface{}]interface{})
+type PrometheusCR struct {
+	Spec struct {
+		Groups []struct {
+			Name  string      `yaml:"name"`
+			Rules []yaml.Node `yaml:"rules"`
+		} `yaml:"groups"`
+	} `yaml:"spec"`
+}
+
+type Rule struct {
+	Expr        string `yaml:"expr"`
+	Record      string `yaml:"record,omitempty"`
+	Alert       string `yaml:"alert,omitempty"`
```

Comment:
It seems to me that we only really make use of Record field. Can we just omit the remaining fields?

Reply:
done